packages/date-time/src/components/WeeklyDayPicker/WeeklyDayPicker.styles.ts

```diff
@@ -0,0 +1,72 @@
+import { IWeeklyDayPickerStyleProps, IWeeklyDayPickerStyles } from './WeeklyDayPicker.types';
+import { normalize, FontSizes, getFocusStyle, getGlobalClassNames } from '@uifabric/styling';
+
+const GlobalClassNames = {
+  root: 'ms-WeeklyDayPicker-root'
+};
+
+export const styles = (props: IWeeklyDayPickerStyleProps): IWeeklyDayPickerStyles => {
+  const { className, theme } = props;
+  const { palette } = theme;
+
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  return {
+    root: [
+      classNames.root,
+      normalize,
+      {
+        width: 196,
+        padding: 12,
+        boxSizing: 'content-box',
+        display: 'flex',
+        flexDirection: 'row'
+      },
+      className
+    ],
+    dayButton: {
+      borderRadius: '100%'
+    },
+    dayIsToday: {},
+    dayCell: {
+      borderRadius: '100%!important'
+    },
+    daySelected: {},
+    navigationIconButton: [
+      getFocusStyle(theme, { inset: -2 }),
+      {
+        width: 12,
+        minWidth: 12,
+        height: 0,
+        overflow: 'hidden',
+        alignSelf: 'flex-end',
+        padding: 0,
+        margin: 0,
+        border: 'none',
+        backgroundColor: palette.neutralLighter,
+        fontSize: FontSizes.small,
+        selectors: {
+          [`.${classNames.root}:hover &, .${classNames.root}:focus-within &, .${classNames.root}:focus &`]: {
```

Comment:
Could this be combined with the &:hover selector? Reason being is when I try to click on just the arrow while my focus is out of the component, it doesn't appear. I have to first hover over the root, then go back to the arrow.